pkg/authenticator/config/config.go

```diff
@@ -150,16 +149,15 @@ func populateConfig() (*Config, error) {
 	}
 
 	// Parse client cert retry count limit if one is provided from env
-	defaultConfig.ClientCertRetryCountLimit = DefaultClientCertRetryCountLimit
-	clientCertRetryCountLimitString := os.Getenv("CONJUR_CLIENT_CERT_RETRY_COUNT_LIMIT")
-	if len(clientCertRetryCountLimitString) > 0 {
-		parsedClientCertRetryCountLimit, err := strconv.Atoi(clientCertRetryCountLimitString)
-		if err != nil {
-			return nil, log.RecordedError(log.CAKC034E, err.Error())
-		}
-
-		defaultConfig.ClientCertRetryCountLimit = parsedClientCertRetryCountLimit
+	clientCertRetryCountLimit, err := utils.IntFromEnvOrDefault(
+		"CONJUR_CLIENT_CERT_RETRY_COUNT_LIMIT",
+		DefaultClientCertRetryCountLimit,
+		nil,
+	)
+	if err != nil {
+		return nil, err
 	}
+	defaultConfig.ClientCertRetryCountLimit = clientCertRetryCountLimit
 
 	return defaultConfig, nil
```

Comment:
if we are updating this, can we change the name of this var? it's _not_ the default config, it's the config that the client will use based on the default config _and_ the user-provided config

Reply:
sure, nice catch.